e2e/content-services/permissions-component.e2e.ts

```diff
@@ -210,4 +213,50 @@ describe('Permissions Component', function () {
 
     });
 
+    describe('Role Site Dropdown', function () {
+
+        beforeEach(async (done) => {
+            await alfrescoJsApi.login(fileOwnerUser.id, fileOwnerUser.password);
+            let siteName = `PUBLIC_TEST_SITE_${Util.generateRandomString(5)}`;
+            let folderName = `MEESEEKS_${Util.generateRandomString(5)}`;
+            let publicSiteBody = { visibility: 'PUBLIC', title: siteName };
+
+            publicSite = await alfrescoJsApi.core.sitesApi.createSite(publicSiteBody);
+
+            uploadedFolder = await uploadActions.createFolder(alfrescoJsApi, folderName, publicSite.entry.guid);
+
+            loginPage.loginToContentServicesUsingUserModel(fileOwnerUser);
+            browser.get(TestConfig.adf.url + '/files/' + publicSite.entry.guid);
```

Comment:
Let's create a method: NavigateToSite() for this line of code.

Reply:
We have too many sites in the UI adfdev environemnt, and it only displays 50-100 at a time , in that small site dropdown. So probably not a very good idea to navigate to site through UI.